docs/source/release_notes.rst

```diff
@@ -45,6 +47,7 @@
     * Fixes
         * Fixed ``TimeSeriesFeaturizer`` to make it deterministic when creating and choosing columns :pr:`3384`
         * Fixed bug where Email/URL features with missing values would cause the imputer to error out :pr:`3388`
+        * Simplified internal ``AutoMLSearch`` API to rely on ``search_parameters`` :pr:`3373`
```

Comment:
the #3373 PR lines should be cleaned up here! line 50 should be removed but unsure why line 41 is here, I see it in main as well.